README.md

```diff
@@ -2606,6 +2609,7 @@ macOS 10.12                | Sierra             | September 20, 2016 | 10.12.6 (
 macOS 10.13                | High Sierra        | September 25, 2017 | 10.13.6 (17G65) (July 9, 2018)
 macOS 10.14                | Mojave             | September 24, 2018 | 10.14.6 (18G3020) (January 28, 2020)
 macOS 10.15                | Catalina           | October 7, 2019    | 10.15.5 (19F101) (June 1, 2020)
+macOS 11.0                 | Big Sur            | February 25, 2021  | 10.2.2 (20D80) (February 25, 2021)
```

Comment:
Thanks, but see point 3 in https://github.com/herrbischoff/awesome-macos-command-line/blob/master/contributing.md.